src/App.scss

```diff
@@ -0,0 +1,75 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
-_-

Reply:
-.-

Reply:
![image](https://user-images.githubusercontent.com/72860434/143852181-59daeddc-7e92-4296-8d9e-ce91b47d4f71.png)

Reply:
I’d call it neither best nor bad practice

Reply:
But since the performance doesn't matter now, it's just easier to reset them like this

Reply:
https://blog.bitsrc.io/using-the-css-star-selector-to-zero-out-margins-and-paddings-bf4c074b83dc

We prefer using reset and normalize files instead. In production code you don't use * selector for that. If that's crucial for you, use it

Reply:
Okay)